pom/pom.xml

```diff
@@ -207,6 +221,84 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>windows-executable</id>
+            <plugins>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-compiler-plugin</artifactId>
+                    <version>3.6.0</version>
+                    <configuration>
+                        <source>${jdkLevel}</source>
+                        <target>${jdkLevel}</target>
+                    </configuration>
+                </plugin>
```

Comment:
Why we have to redefine `maven-compiler-plugin`? Its is already defined in `build / plugins`